huaweicloud/data_source_huaweicloud_cce_node_pool_v3.go

```diff
@@ -0,0 +1,261 @@
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"labels": { //(k8s_tags)
+				Type:     schema.TypeMap,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"root_volume": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"size": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+						"volumetype": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"extend_params": {
+							Type:     schema.TypeMap,
+							Computed: true,
+							Elem:     &schema.Schema{Type: schema.TypeString},
+						},
+					}},
+			},
+			"data_volumes": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"size": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+						"volumetype": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"extend_params": {
+							Type:     schema.TypeMap,
+							Computed: true,
+							Elem:     &schema.Schema{Type: schema.TypeString},
+						},
+					}},
+			},
+			"max_pods": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"extend_param": {
+				Type:     schema.TypeMap,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"availability_zone": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"os": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"key_pair": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"tags": tagsSchema(),
+			"billing_mode": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"subnet_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"scall_enable": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+			"min_node_count": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"max_node_count": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"scale_down_cooldown_time": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"priority": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func dataSourceCceNodePoolsV3Read(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*config.Config)
+	cceClient, err := config.CceV3Client(GetRegion(d, config))
+	if err != nil {
+		return fmt.Errorf("Unable to create HuaweiCloud CCE client : %s", err)
+	}
+
+	listOpts := nodepools.ListOpts{
+		Uid:   d.Get("node_pool_id").(string),
+		Name:  d.Get("name").(string),
+		Phase: d.Get("status").(string),
+	}
+
+	if v, ok := d.GetOk("name"); ok {
+		listOpts.Name = v.(string)
+	}
+
+	if v, ok := d.GetOk("node_pool_id"); ok {
+		listOpts.Uid = v.(string)
+	}
+
+	if v, ok := d.GetOk("status"); ok {
+		listOpts.Phase = v.(string)
+	}
```

Comment:
those lines is duplicated with #L162-L166

Reply:
fixed